unit_tests/test_cluster.py

```diff
@@ -115,6 +116,16 @@ def test_search_system_interlace_reactor_stall(self):
             assert event_b["type"] == "REACTOR_STALLED"
             assert event_b["line_number"] == 3
 
+    def test_search_cdc_invalid_request(self):
+        from sdcm.sct_events.group_common_events import ignore_upgrade_schema_errors
+        self.node.system_log = os.path.join(os.path.dirname(__file__), 'test_data', 'system_cdc_invalid_request.log')
```

Comment:
Why the import is here, is there a cyclic dependency issue here ? If so please comment there is

Reply:
fixed